app/test/e2e/utils.js

```diff
@@ -54,6 +54,211 @@ const createResource = (app = 'rw', vocabularyCount = 1) => {
     };
 };
 
+/**
+ * Mocks the requests for validating a dataset, returning the id of the mock dataset.
+ *
+ * @param {Object} opts Options object
+ * Should contain the nock instance, and (optional) an id to use as the mock dataset's id.
+ *
+ * @returns {String} The id of the mock dataset.
+ */
+const mockDataset = ({ nock, id = undefined }) => {
+    // If id has not been provided, set it as the current timestamp
+    const idToUse = id || new Date().getTime();
```

Comment:
use https://stackoverflow.com/a/17899751/1514576 instead. 
I'm trying to move ID validation into app-land, like so: https://github.com/Skydipper/control-tower/blob/cd90a35c9500f2e264eea65c2d1165959363b171/app/src/plugins/sd-ct-oauth-plugin/services/auth.service.js#L125

weird IDs will fail either on mongo-land or on node-land, and node-land is easier to handle. The "downside" is that our tests should use valid IDs, but that's not too complicated either.